app/js/controllers/relatedCtrl.js

```diff
@@ -20,25 +21,29 @@ KMCMenu.controller('relatedCtrl', ['$scope',
 		// update entries list
 		$scope.entryListChange = function(){
 			// unselect playlist ID
-			$scope.playerData.config.player.plugins["related"].playlistId = null;
+			$scope.playerData.config.plugins["related"].playlistId = null;
+			$scope.playerData.config.plugins["related"].useContext = false;
 			$scope.playlistId = {id:'',text:''};
 			// set entries list
-			$scope.playerData.config.player.plugins["related"].entryList = $scope.entryList;
+			$scope.playerData.config.plugins["related"].entryList = $scope.entryList;
 		};
 
 		// update playlist ID
 		$scope.playlistIdChange = function(){
 			// unselect entries list
-			$scope.playerData.config.player.plugins["related"].entryList = null;
+			$scope.playerData.config.plugins["related"].entryList = [];
+			$scope.playerData.config.plugins["related"].useContext = false;
 			$scope.entryList = "";
 			// set playlist ID
-			$scope.playerData.config.player.plugins["related"].playlistId = $scope.playlistId.id;
+			$scope.playerData.config.plugins["related"].playlistId = $scope.playlistId.id;
 		};
 
 		// clear selection of unselected radio button potions
 		$scope.relatedSelected = function(){
-			$scope.playerData.config.player.plugins["related"].playlistId = undefined;
-			$scope.playerData.config.player.plugins["related"].entryList = null;
+			$scope.playerData.config.plugins["related"].playlistId = null;
+			$scope.playerData.config.plugins["related"].entryList = [];
+			$scope.playerData.config.plugins["related"].useContext = true;
```

Comment:
why true ?

Reply:
relatedSelected() is the  input handler of `$scope.relatedOption = "relatedToEntry";` which is the equivalent of useContext in the plugin config